fpdf/fpdf.py

```diff
@@ -253,6 +253,22 @@ def get_page_format(format, k=None):
         raise FPDFPageFormatException(f"Arguments must be numbers: {args}") from e
 
 
+def get_padding_tuple(padding):
```

Comment:
A suggestion: could you please make this utility function return a `NamedTuple` subclass instead of a `tuple`?
Like this:
```
class Padding(NamedTuple):
    top: Number = 0
    bottom: Number = 0
    left: Number = 0
    right: Number = 0
```

This would make the code a lot clearer, as then you use `padding.top` instead of `padding[0]`